registry/config/config.ini

```diff
@@ -6,10 +6,10 @@
 host=0.0.0.0
 
 # Port for host, overwritten by ENV $APP_PORT
-port=8080
+port=9050
 
 # Database hostname, overwritten by ENV $DB_HOST
-db_host=localhost
+db_host=0.0.0.0

```

Comment:
please don't modify default config, this is deployment specific and can be overwritten